Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -215,6 +215,16 @@ public static class Utils {
 
     #endregion
 
+    #region String Utils
+    public static string TrimEnd(this string str, int characters) {
+      return str.Substring(0, Mathf.Max(0, str.Length - characters));
+    }
+
+    public static string TrimStart(this string str, int characters) {
+      return str.Substring(characters);
```

Comment:
Would be good to have a bounds clamp here too, like in TrimEnd

Reply:
I think this was you tho?

Reply:
Well

Then I will push a fix :D